File: 1878-check-if-array-is-sorted-and-rotated/1878-check-if-array-is-sorted-and-rotated.cpp
class Solution {
public:
    bool check(vector<int>& nums) {
        int pnt = 0 ;
        for(int i = 1 ; i < nums.size() ; i++){
            if(nums[i-1] > nums[i])
                pnt++;
        } 
        if(nums[0] >= nums[nums.size()-1] && pnt == 1)
            return 1 ;
        if(pnt == 0)
            return 1 ;
         else
            return 0 ; 
    }
};

File: 1878-check-if-array-is-sorted-and-rotated/1878-check-if-array-is-sorted-and-rotated.java
class Solution {
    public boolean check(int[] nums) {
        int breakPoint = 0 ;
        for(int i = 1 ; i < nums.length ; i++){
            if(nums[i-1] > nums[i] )
                breakPoint++ ;
        }
        if(nums[0] < nums[nums.length-1]) breakPoint++;
        if(breakPoint > 1)
            return false ;
        return true;
    }
}